packages/flutter_tools/test/commands.shard/hermetic/install_test.dart

```diff
@@ -33,7 +33,7 @@ void main() {
 
       await createTestCommandRunner(command).run(<String>['install']);
     }, overrides: <Type, Generator>{
-      Cache: () => Cache.test(),
+      Cache: () => Cache.test(processManager: FakeProcessManager.any()),
```

Comment:
Does Cache.test() ever need a different ProcessManager?

Reply:
It shouldn't, the process manager is only used for chmod.